src/board/Board.test.jsx

```diff
@@ -0,0 +1,81 @@
+import { screen, render, fireEvent } from "@testing-library/react";
+import Board from "./Board";
+
+test("has the correct number of rows", () => {
+  render(<Board changeTurn={jest.fn()} currentPlayer={"x"} />);
+
+  expect(screen.getByText("1")).toBeVisible();
+  expect(screen.getByText("2")).toBeVisible();
+  expect(screen.getByText("3")).toBeVisible();
+  expect(screen.getByText("4")).toBeVisible();
+  expect(screen.getByText("5")).toBeVisible();
+  expect(screen.getByText("6")).toBeVisible();
+  expect(screen.getByText("7")).toBeVisible();
+  expect(screen.getByText("8")).toBeVisible();
+  expect(screen.getByText("9")).toBeVisible();
+});
+
+describe("when user clicks a cell", () => {
+  test("board is updated with correct marker", () => {
```

Comment:
Small detail but I'd maybe adjust some of the spacing here to improve readability for the next developer. Every team/test is different but I generally like to keep my "action" (e.g. the .click() in a separate block) from the expect and/or the setup. It depends how many lines of code is already in the test / how readable it already is. for example, for this test, i might write this as: 

```   
test("board is updated with correct marker", () => {
  render(
      <Board
        showAlert={jest.fn()}
        closeAlert={jest.fn()}
        changeTurn={jest.fn()}
        currentPlayer="z"
      />
    );
   
   const cellOne = screen.getByText("1");
   fireEvent.click(cellOne);

   expect(screen.queryByText("1")).not.toBeInTheDocument();
   expect(cellOne).toHaveTextContent("z");
})
```
Not a huge deal; it just felt off that `const cellOne = screen.getByText("1");` was isolated and it drew my attention to that line

Also, you don't need variables for every mocked function (e.g. in your props) unless you're planning on asserting it gets called, or to add some clarity with the variable names.